### PR TITLE
feat: [FFM-5219]: Java SDK one click code example for FF Onboarding

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

New one click code example for the Java SDK in the FF onboarding flow.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -564,86 +564,121 @@ onboarding:
...
+      ```
+      <dependency>
+          <groupId>io.harness</groupId>
+          <artifactId>ff-java-server-sdk</artifactId>
+          <version>1.1.6</version>
+      </dependency>
       ```
 
-      `target` represents the desired target for which we want features to be evaluated.
+      Gradel
```
Comment (stephenmcconkey) [inline src/modules/75-cf/strings/strings.en.yaml:593]: typo - Gradel should be Gradle

Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:593]: Daaang, good spot ;) thank you!

File: src/modules/75-cf/strings/strings.en.yaml [hunk 2]
```diff
@@ -564,86 +564,121 @@ onboarding:
...
-      `target` represents the desired target for which we want features to be evaluated.
+      Gradel 
+      ```
+      implementation group: 'io.harness', name: 'ff-java-server-sdk', version: '1.1.6'
+      ```
 
-      Your Harness SDK is now initialized. Congratulations!
+      Create a class called 'GettingStarted' in your project, copy and paste the code below into that class. 
+      After installing the SDK, enter the SDK keys that you created for your environment. 
+      The SDK keys authorize your application to connect to the FF client.
```
Comment (crystal-fong) [inline src/modules/75-cf/strings/strings.en.yaml:600]: `FF client` *chefs kiss*

**Review by stephenmcconkey — APPROVED** [review]
Looks Gucci!

File: src/modules/75-cf/strings/strings.en.yaml [hunk 3]
```diff
@@ -564,86 +564,121 @@ onboarding:
...
-                          .attributes(new HashMap<String, Object>())
-                          .identifier("user1@example.com")
-                          .build();
+      Maven - add the following Maven dependency in your project's 'pom.xml' file:
+      ```
+      <dependency>
+          <groupId>io.harness</groupId>
+          <artifactId>ff-java-server-sdk</artifactId>
+          <version>1.1.6</version>
+      </dependency>
```
[the remark this answers is not in the document]
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:590]: This is a great shout, thanks James :) will add it in!

**Review by crystal-fong — APPROVED** [review]

**Review by stephenmcconkey — APPROVED** [review]

**Review by jcox250 — APPROVED** [review]
Looks good to me!

**Review by crystal-fong — APPROVED** [review]

**Review by stephenmcconkey — APPROVED** [review]

**Pull request merged**